clients/extension/watch.js

```
@@ -1,58 +1,78 @@
 
-chokidar.watch('src/background').on('change', () => build('background'))
+  proc.stderr.on('data', data => process.stderr.write(data))
 
-chokidar.watch('src/content').on('change', () => build('content'))
+  proc.on('exit', code => {
+    if (code) console.error(`\x1b[31m${label} exited with code ${code}\x1b[0m`)
+  })
+}
+
+const cleanup = () => {
+  children.forEach(proc => proc.kill())
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)